source/features/github-bugs.css

```diff
@@ -1,22 +1,3 @@
-/* Place Releases/Tags selector on the same line as `Draft new release` on releases page #2325 */
-.subnav .d-flex.flex-md-row.flex-justify-between.flex-md-items-center {
-	float: left;
-}
```

Comment:
Test URL: https://github.com/refined-github/refined-github/releases

Outdated selector, and the issue is gone:

![image](https://user-images.githubusercontent.com/46634000/153644415-0f716bca-6a49-4c22-8e6d-2acc4652e861.png)

That said, there's some real Picasso-style layout breakage going on, at around 550px wide:

![image](https://user-images.githubusercontent.com/46634000/153644287-897680a0-66de-4c04-91df-dd394e129f72.png)

This is caused by our "Select tag" dropdown, so it might be worth fixing.